### PR TITLE
Pj/status bar hot fix

### LINKED ISSUE #637: [BUG] StatusBar color doesn't update for landscape mode
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

This was reported by @jfversluis 

image

### Expected Behavior

The status bar looks gourgeous

### Steps To Reproduce

No needed

### Link to public reproduction project repository

N/A

### Environment

```markdown
- .NET MAUI CommunityToolkit:1.3.0
- OS:MacOS
- .NET MAUI: 6.0.400
```

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by pictos**
### Description of Change ###

Added Page.Width and Page.Height BP to trigger the StatusBarColor

 ### Linked Issues ###
 

 - Fixes #637

 ### PR Checklist ###
 
 - [x] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [ ] Has tests (if omitted, state reason in description)
 - [x] Has samples (if omitted, state reason in description)
 - [x] Rebased on top of `main` at time of PR
 - [x] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#contributing-code---best-practices)
 - [ ] Documentation created or updated: https://github.com/MicrosoftDocs/CommunityToolkit/pull/XYZ 

 ### Additional information ###

### CODE REVIEW

File: src/CommunityToolkit.Maui/Extensions/PropertyChangedEventArgsExtensions.shared.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+namespace CommunityToolkit.Maui.Extensions;
+
+static class PropertyChangedEventArgsExtensions
+{
+	[MethodImpl(MethodImplOptions.AggressiveInlining)]
+	public static bool Is(this string propertyName, BindableProperty property) =>
+		propertyName == property.PropertyName;
+
+	[MethodImpl(MethodImplOptions.AggressiveInlining)]
```
Comment (VladislavAntonyuk) [inline src/CommunityToolkit.Maui/Extensions/PropertyChangedEventArgsExtensions.shared.cs:13]: Can we use param and compare all items at once 
properties.All(p=> p.PropertyName==propertyName)

Comment (pictos) [reply src/CommunityToolkit.Maui/Extensions/PropertyChangedEventArgsExtensions.shared.cs:13]: params allocate an array, for small numbers of properties isn't efficient. I believe that was the reason that XF team didn't use it

Comment (VladislavAntonyuk) [reply src/CommunityToolkit.Maui/Extensions/PropertyChangedEventArgsExtensions.shared.cs:13]: In terms of maintainability and ease to use I would prefer params. the performance improvement is so small.
Up to you. Approved

**Review by VladislavAntonyuk — APPROVED** [review]

File: src/CommunityToolkit.Maui/Extensions/PropertyChangedEventArgsExtensions.shared.cs (hunk 1 above)
Comment (pictos) [reply src/CommunityToolkit.Maui/Extensions/PropertyChangedEventArgsExtensions.shared.cs:13]: Well I just copied and pasted it, we shouldn't touch this code anymore (I hope), if so we can add the overload with params

**Pull request merged**